src/python/WMCore/WorkQueue/WorkQueue.py

```diff
@@ -1130,6 +1130,10 @@ def processInboundWork(self, inbound_work=None, throw=False, continuous=False, r
                 if work:
                     self.logger.info('Request "%s" already split - Resuming', inbound['RequestName'])
                 else:
+                    # add additional logging for empty inputs when we expect to create
+                    # new workqueue element, see issue-11784
+                    if not inbound['Inputs']:
```

Comment:
I feel like this is either a) not meaningful; or b) not correct. 
One example is MonteCarlo workflow from scratch, which will not have any `Inputs` and that's exactly how it's supposed to be.

Reply:
If such information is not logged yet, maybe just stick with an INFO logging record then.